packages/app/src/client/views/ring-health/RingTable.tsx

```diff
@@ -123,13 +124,19 @@ const RingTable = ({ ringEndpoint, baseUrl }: Props) => {
   const fetchShards = async () => {
     try {
       const response = await axios.get<Payload>(ringEndpoint);
+      if (response.headers["content-type"] !== "application/json") {
+        // some error messages are returned via HTML
+        // So we throw an error here to trigger an error toast
```

Comment:
For example [here](https://opstrace-community.slack.com/archives/C01KZS8JRNC/p1623664061091000)

Reply:
We should at least populate the Error with an appropriate message

Reply:
It would fall back to the current generic message [here](https://github.com/opstrace/opstrace/pull/876/files#diff-4d85fd5df3e23c5aa25f39c3f608b4cd7e4baf23c713929e440dc7e1701468d0R155).
Should we add a different message for this use case?

Reply:
I think we should display the error message to the user

Reply:
Let's do that in a follow up PR though and get this one landed!